_source/_posts/2021-06-18-native-java-framework-comparison.adoc

```diff
@@ -320,12 +303,12 @@ __  ____  __  _____   ___  __ ____  ______
  --/ __ \/ / / / _ | / _ \/ //_/ / / / __/
  -/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \
 --\___\_\____/_/ |_/_/|_/_/|_|\____/___/
-2021-09-15 09:42:35,854 INFO  [io.quarkus] (main) quarkus 1.0.0-SNAPSHOT native (powered by Quarkus 2.2.2.Final) started in 0.018s. Listening on: http://0.0.0.0:8080
-2021-09-15 09:42:35,855 INFO  [io.quarkus] (main) Profile prod activated.
-2021-09-15 09:42:35,855 INFO  [io.quarkus] (main) Installed features: [cdi, resteasy, security, smallrye-context-propagation, smallrye-jwt, vertx, vertx-web]
+2021-10-06 23:19:46,957 INFO  [io.quarkus] (main) quarkus 1.0.0-SNAPSHOT native (powered by Quarkus 2.3.0.Final) started in 0.022s. Listening on: http://0.0.0.0:8080
+2021-10-06 23:19:46,957 INFO  [io.quarkus] (main) Profile prod activated.
+2021-10-06 23:19:46,957 INFO  [io.quarkus] (main) Installed features: [cdi, reactive-routes, resteasy, security, smallrye-context-propagation, smallrye-jwt, vertx]
```

Comment:
Changing `quarkus-resteasy*` with `quarkus-resteasy-reactive*` should require no other change and result in smaller runtime footprint

Reply:
Excellent, thanks 👍